release.sh

```diff
@@ -0,0 +1,32 @@
+#!/bin/bash
+
+# Find all package.json files in subdirectories, except node_modules, examples and irma-frontend
+set -euxo pipefail
+standalone_packages=`find . -name "package.json" -not -path "*/node_modules/*" -not -path "*/examples/*" -not -path "*/irma-frontend/*"`
+set +euxo pipefail
+
+echo ""
+echo "Are you sure you want to release a new version of the following packages?"
+for package in ${standalone_packages[@]}; do
+  echo " - `dirname $package`"
+done
+read -p "(Enter y/n) "
+
+if [[ $REPLY =~ ^[Yy]$ ]]
+then
+  set -euxo pipefail
+
+  root=`pwd`
+  for package in ${standalone_packages[@]}; do
+    cd `dirname $package`
+    echo "Releasing $package"
+    npm publish --access public
+    cd $root
+  done
+
+  set +euxo pipefail
+
+  echo ""
+  echo "Releasing done."
+  echo "Don't forget releasing irma-frontend separately by running ./release-irma-frontend.sh"
```

Comment:
This comment is confusing, because the mentioned script does not actually release irma-frontend.

Reply:
When the script is renamed, this should be renamed too indeed.